ydb/core/blobstorage/dsproxy/dsproxy_request_reporting.cpp

```diff
@@ -0,0 +1,50 @@
+#include "dsproxy_request_reporting.h"
+
+namespace NKikimr {
+
+static std::array<std::atomic<bool>, NKikimrBlobStorage::EPutHandleClass_MAX> ReportPutPermissions;
+static std::array<std::atomic<bool>, NKikimrBlobStorage::EGetHandleClass_MAX> ReportGetPermissions;
+
+bool AllowToReport(NKikimrBlobStorage::EPutHandleClass handleClass) {
+    return ReportPutPermissions[(ui32)handleClass - 1].exchange(false);
```

Comment:
At least check boundaries. What is we add another handle class in the future? Why (-1)? Saving extra 8 bytes of static process memory?